homework_1/task_03/pokemon/pokemon.go

```diff
@@ -0,0 +1,75 @@
+package pokemon
+
+import (
+	"encoding/json"
+	"github.com/pkg/errors"
+	"github.com/sethgrid/pester"
+	"io/ioutil"
+)
+
+const pokemonAPI = "https://pokeapi.co/api/v2/pokemon/"
+
+type Location struct {
+	Name string
+}
+
+type Encounter struct {
+	Location Location `json:"location_area"`
+}
+
+type Pokemon struct {
+	Name string `json:"name"`
+}
+
+type Output struct {
+	Name string
+	Locations []string
+}
+
+func GetData(url string) ([]byte, error) {
+	httpClient := pester.New()
+
+	httpResponse, err := httpClient.Get(url)
```

Comment:
Using `pester` here is a serious overkill. I would recommend simply using HTTP client implementation from the standard library. You can even do something like this:
```
import "net/http"
[...]
httpResponse, err := http.Get(url)
```